src/color-selector.js

```diff
@@ -126,110 +127,113 @@
         // + '#virtualclassCont.congrea #virtualclassAppContainer .virtualclass .commandToolsWrapper .tool ul.strkSizeList li.selected,'
         // + '#virtualclassCont.congrea #virtualclassAppContainer .virtualclass .commandToolsWrapper .tool ul.fontSizeList span.selected,'
         + '#virtualclassCont.congrea #playButton,'
         + '#virtualclassCont.congrea #recordPlay .rv-vanilla-modal-body #downloadPcCont #downloadSessionText'
-        + `{background-image: linear-gradient(to bottom, ${allbg.fcolor} 0%,${allbg.scolor} 100%) !important} `
+        + `{background-image: linear-gradient(to bottom, ${allbg.fcolor} 0%,${allbg.scolor} 100%)} `
 
 
         + '#virtualclassCont.congrea #virtualclassOptionsCont:first-child '
         + `{background-image: linear-gradient(to right, ${allbg.fcolor} 0%,${allbg.scolor} 100%)} `
 
         + '#virtualclassCont.congrea #virtualclassOptionsCont .appOptions.active '
-        + `{background-image: radial-gradient(ellipse at center, ${active.fcolor} 0%,${active.scolor} 100%) !important;} `
+        + `{background-image: radial-gradient(ellipse at center, ${active.fcolor} 0%,${active.scolor} 100%)} `
         + '#virtualclassCont.congrea .tool.active a span:before '
-        + `{color:${frontColor}!important}`
+        + `{color:${frontColor}}`
         + '#virtualclassCont.congrea #virtualclassOptionsCont .appOptions:hover, '
         + '#virtualclassCont.congrea .containerWb .commandToolsWrapper .tool a:hover, '
         + '#virtualclassCont.congrea .containerWb .commandToolsWrapper .tool ul.fontSizeList span:hover, '
         + '#virtualclassCont.congrea.orginalTeacher #virtualclassAppContainer .virtualclass .commandToolsWrapper .tool[data-tool="stroke"] ul.strkSizeList li:hover, '
         + '#virtualclassCont.congrea #dashboardContainer .modal-header button.enable:hover, '
         + '#virtualclassCont.congrea #confirmCancel #confirmCancelButton:hover, '
         + '#virtualclassCont.congrea #confirm.popupWindow #confirmOk #confirmOkButton:hover, '
-        + '#virtualclassCont.congrea #popupContainer #about-modal #infoMessage #infoMessageOk:hover '
-        + `{background-image: radial-gradient(ellipse at center, ${hover.fcolor} 0%,${hover.scolor} 100%) !important}`
+        + '#virtualclassCont.congrea #popupContainer #about-modal #infoMessage #infoMessageOk:hover, '
+        + '#virtualclassCont.congrea a.vceditor-btn:hover, '
+        + '#virtualclassCont.congrea .vceditor-dropdown-menu a:hover, '
+        + '#virtualclassCont.congrea #virtualclassEditorRich .vceditor-toolbar-wrapper .vceditor-btn-style-group:hover, '
+        + '#virtualclassCont.congrea #virtualclassEditorRich .vceditor-toolbar-wrapper .vceditor-btn-list-group:hover, '
+        + '#virtualclassCont.congrea #virtualclassEditorRich .vceditor-toolbar-wrapper .vceditor-btn-indent-group:hover, '
+        + '#virtualclassCont.congrea #virtualclassEditorRich .vceditor-toolbar-wrapper .vceditor-btn-paragraph-group:hover '
```

Comment:
Codacy found an issue: [This line has a length of 124. Maximum allowed is 120.](https://app.codacy.com/gh/congrea/virtualclass/pullRequest?prid=5740126)